Sources/PackagePlugin/PackageModel.swift

```diff
@@ -418,6 +418,13 @@ extension FileList: Sequence {
     }
 }
 
+@available(_PackageDescription, introduced: 999.0)
```

Comment:
probably can be set to 5.10

Reply:
Just doing what @neonichu asked for.  Can you two sort out what you want and tell me?

Reply:
@neonichu at this point this can be 5.10, no?

Reply:
Yah, my previous comment was from August when 5.10 hadn't been announced yet, now it can be 5.10